modules/internal/ChapelRange.chpl

```diff
@@ -29,6 +29,8 @@ module ChapelRange {
   config param useOptimizedRangeIterators = true;
 
   enum BoundedRangeType { bounded, boundedLow, boundedHigh, boundedNone };
+
+  proc idx2strType(type idxType) type  return chpl__signedType(idxType);
```

Comment:
I'm not hugely opposed to it, but it took me a minute to realize what the proc name stood for. I don't think indexToStrType is too much longer that it'd be annoying.